ember/app/ui/components/identity-emails/template.hbs

```diff
@@ -0,0 +1,123 @@
+<div
+  {{did-insert this.onUpdate}}
+  {{did-update this.onUpdate @identity}}
```

Comment:
Is this really the modern way to write this? AFAIK the modifiers should be used sparingly, and this one looks like it might lead to overfetching. cc @anehx

Reply:
I don't really know anymore since I'm not really writing this kind of code. I think this way it will definitely trigger the fetching too much. At the moment, I'd try keeping all of the fetching in the controller with something like [this](https://github.com/kellyselden/ember-controller-lifecycle/blob/master/addon/initializers/controller-lifecycle.js) or doing it directly from the route.